pkgs/applications/editors/neovim/neovide/default.nix

```diff
@@ -31,16 +31,20 @@
 }:
 rustPlatform.buildRustPackage rec {
   pname = "neovide";
-  version = "0.10.1";
+  version = "0.10.2";
 
   src = fetchFromGitHub {
     owner = "Kethku";
     repo = "neovide";
     rev = version;
-    sha256 = "sha256-PViSiK6+H79MLIOFe26cNqUZ6gZdqDC/S+ksTrbOm54=";
+    postFetch = ''
```

Comment:
please use [`cargoPatches`](https://nixos.org/manual/nixpkgs/unstable/#building-a-crate-with-an-absent-or-out-of-date-cargo.lock-file) instead

Reply:
Thanks, I updated the PR.